compiler/rustc_hir_analysis/src/check/compare_impl_item.rs

```diff
@@ -35,6 +35,26 @@ use crate::errors::{LifetimesOrBoundsMismatchOnTrait, MethodShouldReturnFuture};
 
 mod refine;
 
+/// Call the query `tcx.compare_impl_item()` directly instead.
+pub(super) fn compare_impl_item(
```

Comment:
move this into a submodule so that we don't need to rename them to `_inner`

Reply:
i kinda just renamed them inner for style reasons, but i could just also revert their names

Reply:
hmm, then yeah, I would like to revert their naming then :thinking: